Makefile.js

```diff
@@ -60,7 +61,7 @@ var NODE = "node ", // intentional extra space
     MAKEFILE = "./Makefile.js",
     PACKAGE = "./package.json",
     JS_FILES = find("lib/").filter(fileType("js")).join(" "),
-    JSON_FILES = find("conf/").filter(fileType("json")).join(" ") + " .eslintrc",
+    JSON_FILES = find("conf/").filter(fileType("json")),
```

Comment:
@ilyavolodin `.eslintrc` file does not exist. Your earlier JSON validation script was flawed. I made a comment about this in one of the earlier commit amendments.